pkg/steps/pod_test.go

```diff
@@ -3,7 +3,7 @@ package steps
 import (
 	"testing"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
I think this was vscode auto-resolving something. Can undo if needed.

Reply:
Please undo.

Reply:
Will do